third_party/musl/crt/patch.diff

```diff
@@ -211,7 +211,53 @@ index afec985a..5435b9b5 100644
  #define MAXP2(a,b) (-(-(a)&-(b)))
  #define ALIGN(x,y) ((x)+(y)-1 & -(y))
  
-@@ -1043,38 +1050,7 @@ static struct dso *load_library(const char *name, struct dso *needed_by)
+@@ -217,6 +224,14 @@ static int search_vec(size_t *v, size_t *r, size_t key)
+ 	return 1;
+ }
+ 
++static size_t* search_vec2(size_t *v, size_t *r, size_t key)
```

Comment:
Suggest renaming to `find_dynv_value_addr_by_key`

Reply:
Renamed.